index.js

```diff
@@ -17,9 +17,14 @@ const { preLoadPullRequestsAndIssues } = require('./statistics/pre_load')
 
 module.exports = (app) => {
   console.log("Creating tables if not exist");
-  creatTablesIfNotExist(app).then(
-      () => console.log("Tables created")
-  );
+
+  app.onAny(async (context) => {
```

Comment:
The creation of database should only run once at initialization. onAny makes it run on every event. I still think we should run it once even in serverless environment.

Reply:
>once at initialization

What do you mean by `initialization`?

In a serverless environment, all the runs are same, there is not static machine, where the code run. Alternatively you can just run it once manually and remove this from lambda.

Reply:
Although it's no difference in serverless environment but if we run it as a server, we don't want to check every time an event come in. We only need to check if table exists when the app starts.

Reply:
> Although it's no difference in serverless environment but if we run it as a server, we don't want to check every time an event come in. We only need to check if table exists when the app starts.

I am not sure, what action are you suggesting here (?)

Reply:
Move it out of onAny and I'll make creatTablesIfNotExist non-async.

Reply:
It's only make the code look better. Let's keep it that way. I'll merge this PR.